packages/backend/src/controllers/collection.controller.ts

```diff
@@ -271,3 +272,52 @@ export async function deleteTradeOffer(req: Request, res: Response) {
   }
 }
 
+// View Comic Book Offers
+
+export async function viewTradeOffers(req: Request, res: Response) {
+  try {
+    const tradeOffers = await viewComicBookOffers();
+
+    if (!tradeOffers.length) {
+      return res
+        .status(404)
```

Comment:
if there are 0 trade offers can we return an empty array and 200 or 201 status?
404  is resource not found. In this case it is found, it's just empty.